controllers/front/DispatchWebHook.php

```diff
@@ -36,9 +36,9 @@ class ps_checkoutDispatchWebHookModuleFrontController extends ModuleFrontControl
     public $auth = false;
 
     /**
-     * Id coming from PSL
+     * Id coming from PSL (CheckMe: Is it a UUID?)
```

Comment:
@Matt75 / @v4lux, can you please confirm this property stores a string?

Reply:
Yes, it's a string